asistente_ladm_col/logic/quality_rules/qr_parcel_with_invalid_parcel_number.py

```diff
@@ -0,0 +1,105 @@
+"""
+/***************************************************************************
+                              Asistente LADM_COL
+                             --------------------
+        begin                : 2022-05-04
+        git sha              : :%H$
+        copyright            : (C) 2022 by Leo Cardona (BSF Swissphoto)
+                               (C) 2022 by Sergio Ramírez (BSF Swissphoto)
+        email                : leo.cardona.p@gmail.com
+                               seralra96@gmail.com
+
+ ***************************************************************************/
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License v3.0 as          *
+ *   published by the Free Software Foundation.                            *
+ *                                                                         *
+ ***************************************************************************/
+"""
+from qgis.PyQt.QtCore import (QCoreApplication,
+                              QSettings)
+
+from asistente_ladm_col.config.enums import (EnumQualityRuleType,
+                                             EnumQualityRuleResult)
+from asistente_ladm_col.config.keys.common import (QUALITY_RULE_LAYERS,
+                                                   QUALITY_RULE_LADM_COL_LAYERS,
+                                                   QUALITY_RULE_ADJUSTED_LAYERS,
```

Comment:
We're not using all of these constants, could you check and delete what is not in use?

Reply:
thanks for the review, adjustment made in the commit [31737a7](https://github.com/SwissTierrasColombia/Asistente-LADM-COL/commit/31737a7836f67507d9e41164dce8ecc6972d1638)